src/main/haskell/kore/app/exec/Main.hs

```diff
@@ -126,6 +128,12 @@ commandLineParser =
     <*> enableDisableFlag "is-program"
         True False False
         "Whether the pattern represents a program."
+    <*> option auto
+        ( metavar "SMT_TIMEOUT"
+        <> long "smt_timeout"
```

Comment:
It's conventional to us `-` in long command-line options, so this should be `smt-timeout` rather than `smt_timeout`.

What would happen if the user passed a negative time-out?